lib/Event/Model/Ecommerce/IndexService/PreprocessErrorEvent.php

```diff
@@ -27,17 +27,23 @@ class PreprocessErrorEvent extends Event
      * @var bool
      */
     protected $throwException;
-
+    
+    /**
+     * @var int
+     */
+    protected $subObjectId;
+    
     /**
      * PreprocessErrorEvent constructor.
      *
      * @param \Throwable $exception
      * @param bool $throwException
      */
-    public function __construct(\Throwable $exception, bool $throwException = true)
+    public function __construct(\Throwable $exception, bool $throwException = true, $subObjectId = 0)
```

Comment:
please adapt the phodoc

Reply:
Should the parameter be typed?
```suggestion
    public function __construct(\Throwable $exception, bool $throwException = true, int $subObjectId = 0)
```

Reply:
thanks!